db/api/v1/db/assets.db.js

```diff
@@ -20,10 +18,6 @@ const find = async (projectId) => {
     let values;
 
     if (typeof projectId !== 'undefined') {
-        if (!utils.db.isValidDbInteger(projectId)) {
```

Comment:
Should we have checking in each component because they "could be" used independently from controllers? Or is that too much redundancy?